llvm/include/llvm/CodeGen/GlobalISel/MachineIRBuilder.h

```diff
@@ -856,8 +856,9 @@ class MachineIRBuilder {
   ///      type.
   ///
   /// \return The newly created instruction.
-  virtual MachineInstrBuilder buildConstant(const DstOp &Res,
-                                            const ConstantInt &Val);
+  virtual MachineInstrBuilder
+  buildConstant(const DstOp &Res, const ConstantInt &Val,
+                SmallVectorImpl<DILocation *> *Locs = nullptr);
```

Comment:
Do we really have to pollute the public API with this? Can't this remain an internal implementation detail?

Reply:
Good point! I will make the change